test/protocol/PriceDiscoveryHandlerFacet.js

```diff
@@ -504,7 +506,23 @@ describe("IPriceDiscoveryHandlerFacet", function () {
               priceDiscoveryHandler
                 .connect(buyer)
                 .commitToPriceDiscoveryOffer(buyer.address, tokenId, priceDiscovery, { value: price })
-            ).to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED);
+            )
+              .to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED)
+              .withArgs(PausableRegion.Buyers);
+          });
+
+          it("The price discovery region of protocol is paused", async function () {
+            // Pause the price discovery region of the protocol
+            await pauseHandler.connect(pauser).pause([PausableRegion.PriceDiscovery]);
+
+            // Attempt to sequentially commit, expecting revert
+            await expect(
+              priceDiscoveryHandler
+                .connect(buyer)
+                .commitToPriceDiscoveryOffer(buyer.address, tokenId, priceDiscovery, { value: price })
+            )
+              .to.revertedWithCustomError(bosonErrors, RevertReasons.REGION_PAUSED)
+              .withArgs(PausableRegion.PriceDiscovery);
           });
 
```

Comment:
Could be interesting to test that commit to a fixed price offer is still possible, even when the PriceDiscovery region is Paused

Reply:
Added (ouside "revert reasons" context).